`lerna run` choked on Node v14.17.5

### Describe the bug

Executing `lerna run` on Node v14.17.5 runtime of `pnpm` gave me:
```shell
➜  fiddle git:(pnpm) pnpm node -v
v14.17.5
➜  fiddle git:(pnpm) pnpm x

> root@ x /Users/i532660/github/fiddle
> lerna run x

internal/modules/cjs/loader.js:892
  throw err;
  ^

Error: Cannot find module 'node:fs'
Require stack:
- /Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+version@1.6.0/node_modules/@lerna-lite/version/dist/lib/update-lockfile-version.js
- /Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+version@1.6.0/node_modules/@lerna-lite/version/dist/lib/index.js
- /Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+version@1.6.0/node_modules/@lerna-lite/version/dist/index.js
- /Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+cli@1.6.0/node_modules/@lerna-lite/cli/dist/cli-commands/cli-version-commands.js
- /Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+cli@1.6.0/node_modules/@lerna-lite/cli/dist/cli-commands/cli-publish-commands.js
- /Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+cli@1.6.0/node_modules/@lerna-lite/cli/dist/lerna-entry.js
- /Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+cli@1.6.0/node_modules/@lerna-lite/cli/dist/cli.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.<anonymous> (/Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+version@1.6.0/node_modules/@lerna-lite/version/dist/lib/update-lockfile-version.js:10:19)
    at Module._compile (internal/modules/cjs/loader.js:1072:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1101:10)
    at Module.load (internal/modules/cjs/loader.js:937:32)
    at Function.Module._load (internal/modules/cjs/loader.js:778:12)
    at Module.require (internal/modules/cjs/loader.js:961:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+version@1.6.0/node_modules/@lerna-lite/version/dist/lib/update-lockfile-version.js',
    '/Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+version@1.6.0/node_modules/@lerna-lite/version/dist/lib/index.js',
    '/Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+version@1.6.0/node_modules/@lerna-lite/version/dist/index.js',
    '/Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+cli@1.6.0/node_modules/@lerna-lite/cli/dist/cli-commands/cli-version-commands.js',
    '/Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+cli@1.6.0/node_modules/@lerna-lite/cli/dist/cli-commands/cli-publish-commands.js',
    '/Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+cli@1.6.0/node_modules/@lerna-lite/cli/dist/lerna-entry.js',
    '/Users/i532660/github/fiddle/node_modules/.pnpm/registry.npmjs.org+@lerna-lite+cli@1.6.0/node_modules/@lerna-lite/cli/dist/cli.js'
  ]
}
 ELIFECYCLE  Command failed with exit code 1.
```

Running on a higher version, e.g. v16.16.0, seemed to be fine.

### Expectation

_No response_

### Reproduction

Should be reproducible here:
https://github.com/dwiyatci/fiddle/tree/pnpm

Simply run `pnpm x` :-)

### Lerna config and logs

<details><summary>lerna.json</summary><p>
<!-- browsers demand the next line be empty -->

```json
<!-- Please paste your `lerna.json` here -->
```
</p></details>

<details><summary>lerna-debug.log</summary><p>
<!-- browsers demand the next line be empty -->

```txt
<!-- If you have a `lerna-debug.log` available, please paste it here -->
<!-- Otherwise, feel free to delete this <details> block -->
```
</p></details>


### Environment Info

```shell
Environment info:

  System:
    OS: macOS 12.4
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 14.17.5 - ~/Library/pnpm/node
    Yarn: 1.19.1 - ~/.volta/bin/yarn
    npm: 6.14.14 - ~/Library/pnpm/npm
  Utilities:
    Git: 2.32.1 - /usr/bin/git
```


### Used Package Manager

pnpm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/ghiscoding/lerna-lite/blob/master/.github/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://github.com/ghiscoding/lerna-lite#readme).
- [X] Check that there isn't [already an issue](https://github.com/ghiscoding/lerna-lite/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/ghiscoding/lerna-lite/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

--- a/packages/version/src/__tests__/update-lockfile-version.spec.ts
+++ b/packages/version/src/__tests__/update-lockfile-version.spec.ts
@@ -7,7 +7,7 @@ jest.mock('@lerna-lite/core', () => ({
 import path from 'path';
 import fs from 'fs-extra';
 import core, { Package } from '@lerna-lite/core';
-import nodeFs from 'node:fs';
+import nodeFs from 'fs';
 import npmlog from 'npmlog';
 
 // mocked or stubbed modules

--- a/packages/version/src/__tests__/version-bump.spec.ts
+++ b/packages/version/src/__tests__/version-bump.spec.ts
@@ -1,4 +1,4 @@
-const nodeFs = require('node:fs');
+import nodeFs from 'fs';
 jest.spyOn(nodeFs, 'renameSync');
 
 // local modules _must_ be explicitly mocked

--- a/packages/version/src/__tests__/version-ignore-changes.spec.ts
+++ b/packages/version/src/__tests__/version-ignore-changes.spec.ts
@@ -1,4 +1,4 @@
-const nodeFs = require('node:fs');
+import nodeFs from 'fs';
 jest.spyOn(nodeFs, 'renameSync');
 
 // local modules _must_ be explicitly mocked

--- a/packages/version/src/lib/update-lockfile-version.ts
+++ b/packages/version/src/lib/update-lockfile-version.ts
@@ -1,7 +1,7 @@
 import log from 'npmlog';
 import path from 'path';
 import loadJsonFile from 'load-json-file';
-import { promises as fsPromises, renameSync } from 'node:fs';
+import fs from 'fs';
 import os from 'os';
 import semver from 'semver';
 import writeJsonFile from 'write-json-file';
@@ -168,7 +168,7 @@ export async function runInstallLockFileOnly(
 
           // 2. rename "npm-shrinkwrap.json" back to "package-lock.json"
           log.verbose('lock', `renaming "npm-shrinkwrap.json" file back to "package-lock.json"`);
-          renameSync('npm-shrinkwrap.json', 'package-lock.json');
+          fs.renameSync('npm-shrinkwrap.json', 'package-lock.json');
         }
 
         outputLockfileName = inputLockfileName;
@@ -195,7 +195,7 @@ export async function runInstallLockFileOnly(
  */
 export async function validateFileExists(filePath: string) {
   try {
-    await fsPromises.access(filePath);
+    await fs.promises.access(filePath);
     return true;
   } catch {
     return false;

